pkg/operator/resources/operator/operator.go

```diff
@@ -74,41 +80,45 @@ func getClusterPolicyRules() []rbacv1.PolicyRule {
 		},
 		{
 			APIGroups: []string{
-				"",
+				"apiextensions.k8s.io",
 			},
 			Resources: []string{
-				"pods",
-				"services",
+				"customresourcedefinitions",
+				"customresourcedefinitions/status",
 			},
 			Verbs: []string{
 				"get",
 				"list",
 				"watch",
+				"create",
+				"update",
 				"delete",
 			},
 		},
 		{
 			APIGroups: []string{
-				"apiextensions.k8s.io",
+				"cdi.kubevirt.io",
+				"upload.cdi.kubevirt.io",
 			},
 			Resources: []string{
-				"customresourcedefinitions",
-				"customresourcedefinitions/status",
+				"*",
```

Comment:
nit:
why *

Reply:
All CDI resources (datavolumes/datasources/..)

Reply:
As far as i know the operator doesn't create or delete datavolumes/datasources/..

Reply:
Oh sorry i missed that :+1: